<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="LAPTOP-T4UG6R5F" failures="1" tests="3" name="Test" time="354.342" errors="0" timestamp="2022-12-18T00:32:13 IST">
  <testcase classname="com.eduSol.runner.TestNGRunner_extent_reporeter" name="testcase" time="14.381"/>
  <testcase classname="com.eduSol.runner.TestNGRunner_extent_reporeter" name="testcase" time="31.367">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [true] but found [false]
at com.eduSol.pom.PIM_Login.addemployee(PIM_Login.java:66)
at com.eduSol.runner.TestNGRunner_extent_reporeter.testcase(TestNGRunner_extent_reporeter.java:74)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- testcase -->
  <testcase classname="com.eduSol.runner.TestNGRunner_extent_reporeter" name="testcase" time="0.005">
    <skipped/>
  </testcase> <!-- testcase -->
  <testcase classname="com.eduSol.runner.TestNGRunner_extent_reporeter" name="t2" time="0.0">
    <ignored/>
  </testcase> <!-- t2 -->
</testsuite> <!-- Test -->
